app/forms/mappings/Constraints.scala

```diff
@@ -23,58 +23,81 @@ import java.time.{LocalDate, LocalDateTime, Period}
 
 trait Constraints {
 
-  private lazy val etmpStatementsDate: LocalDate = LocalDate.of(2019, 10, 1)
-  private lazy val pvatStatementsDate: LocalDate = LocalDate.of(2021, 1, 1)
-  private lazy val dutyDefermentStatementsDate: LocalDate = LocalDate.of(2019, 9, 1)
-  private val olderThan = Period.ofMonths(6)
+  val offset = 6
+  val one = 1
```

Comment:
Better to create two vals instead of one as there is a context involved and use 

val dayOne = 1. // for the day in all the dates
val pvatStatementMonth = 1  // in pvatStatementsDate for the month value